handler.c

```diff
@@ -1,562 +1,546 @@
 #include <ApplicationServices/ApplicationServices.h>
 #include <CoreFoundation/CoreFoundation.h>
 
-#include <sys/types.h>
-#include <sys/param.h>
-#include <sys/stat.h>
 #include <dirent.h>
 #include <errno.h>
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
+#include <sys/param.h>
+#include <sys/stat.h>
+#include <sys/types.h>
 
 #include "handler.h"
 #include "plist.h"
 #include "util.h"
 
-extern int		verbose;
-extern struct roles	rtm[];
-int			nroles;
-
-static void dump_cf_array( const void *value, void *context );
-static void dump_cf_dictionary( const void *key, const void *value, void *context );
+extern int verbose;
+extern struct roles rtm[];
+int nroles;
+
+static void dump_cf_array(const void *value, void *context);
+static void dump_cf_dictionary(const void *key, const void *value,
```

Comment:
Move second param to its own line, prefer all on one line (if possible) or one per line.